### PR TITLE
exclude _attachments from encryption

### LINKED ISSUE #17: Not working in chrome
I was trying to use this modul in my angular based web app but keeps on getting 

Uncaught (in promise) RangeError:
Trying to access beyond buffer length(…)
checkOffset @ bundle.js:7543
Buffer.readUInt32LE @ bundle.js:7566
Chacha20 @ bundle.js:
8225Cipher @ bundle.js:8089
createCipher @ bundle.js:8189
encrypt @ bundle.js:22238
incoming @ bundle.js:
21795handlers.bulkDocs @ bundle.js:21844

### PR DESCRIPTION
**Pull request by gr2m**
- allows to workaround #13
- fixes #17 (outdated browserify version was the cause)

### CODE REVIEW

**Comment by calvinmetcalf** [comment]
hm so currently this is a bit of a foot gun potentially leaving you exposed, we have a couple other options

1. throw an informative error if somebody tries to upload an attachment
2. delete the attachments by default
3. encrypt them somehow

now 3 would be preferred but are we able to actually do that without also monkey patching [get|set]Attachment

**Comment by xMartin** [comment]
An option to leave attachments unencrypted would be great. Some Privacy and Security Audits are ok with unencrypted binaries (e.g. images) for some reason. But of course just encrypting them is even better. Don't know about the performance implications.

**Comment by nolanlawson** [comment]
Encrypting large binary objects is probably going to be a huge performance drain, especially in browsers. Also there's the awkwardness of translating between Blobs and [Browserified Buffers](https://github.com/feross/buffer).

I think just adding a note to the README that this module doesn't encrypt attachments should be fine. If somebody wants encrypted attachments, they can clone it.

**Comment by xMartin** [comment]
👍

**Comment by gr2m** [comment]
Added a notet to `readme.md` that attachments are not encrypted. Can this be merged now?

**Comment by calvinmetcalf** [comment]
I'm on the fence about this, but I think I'd like you to have to pass an option to get it to leave attachments untouched.  Encryption isn't very helpful if it silently doesn't encrypt your stuff.

I think the default behavior should be to remove attachments and log a warning, but you can pass an option to get it to ignore them.  I have a pull that is waiting on nolanlawson/transform-pouch#19 which simplifies the creation process which should make passing params easier.

**Comment by gr2m** [comment]
"remove attachments" in the sense of "ignore attachments" / not encrypt them? Happy to help review https://github.com/nolanlawson/transform-pouch/pull/19 to, just won’t get to it today

File: index.js [hunk 1]
```diff
@@ -84,6 +88,9 @@ function cryptoInit(password, modP) {
     if (rev) {
       outDoc._rev = rev;
     }
+    if (attachments) {
+      outDoc._attachments = attachments;
```
Comment (calvinmetcalf) [inline index.js]: I was thinking that be default we wouldn't do this

**Comment by gr2m** [comment]
closing in favor of #29

**Pull request closed without merging**